Swiftfin tvOS/Views/SettingsView/ExperimentalSettingsView.swift

```diff
@@ -28,9 +33,19 @@ struct ExperimentalSettingsView: View {
 
 				Toggle("Sync Subtitles with Adjacent Episodes", isOn: $syncSubtitleStateWithAdjacent)
 
+				Toggle("Native Player", isOn: $nativePlayer)
+
+			} header: {
+				L10n.experimental.text
+			}
+
+			Section {
+
 				Toggle("Live TV (Alpha)", isOn: $liveTVAlphaEnabled)
 
-				Toggle("Native Player", isOn: $nativePlayer)
+				Toggle("Live TV Force Direct Play", isOn: $liveTVForceDirectPlay)
+
+				Toggle("Live TV Native Player", isOn: $liveTVNativePlayer)
 
 			} header: {
 				L10n.experimental.text
```

Comment:
Let's just call this section "Live TV" since it's the section for it, I'm fine for not localizing this as it will eventually be moved out.

Reply:
Sure- thought you might catch this :) 
But as it turns out, I'm not sure how to add new localized strings! Seems like there is a generated file, is there a tool I need to use? Or how does that work?